Shared/Features/Freaks/View/FreaksView.swift

```diff
@@ -3,11 +3,28 @@ import SwiftUI
 struct FreaksView: View {
     var viewModel = FreaksViewModel()
 
+    init() {
+        let coloredNavAppearance = UINavigationBarAppearance()
+
+        coloredNavAppearance.titleTextAttributes = [.foregroundColor: UIColor(Color("SecondaryColor"))]
+
+        coloredNavAppearance.backgroundColor = UIColor(Color("AccentColor"))
+
+        UINavigationBar.appearance().standardAppearance = coloredNavAppearance
+
+        UINavigationBar.appearance().scrollEdgeAppearance = coloredNavAppearance
+    }
+
+    var preferredStatusBarStyle: UIStatusBarStyle {
```

Comment:
`preferredStatusBarStyle` setup  is needed anymore? I think after info.plist setup, this may not be applied.